### DISCUSSION TITLE
I hope this tool will support SQL diagnosis, such as slow SQL analysis

### DISCUSSION THREAD

**Discussion by ganggangtongxue** [oceanbase/obdiag#22] [Ideas]
I hope this tool will support SQL diagnosis, such as slow SQL analysis

**Comment by Teingi** [comment]
Yes, we have currently added parallel SQL analysis capabilities, and will consider adding other SQL analysis capabilities in the future, such as Slow SQL.

**Comment by Teingi** [comment]
有2种方式可以去实现：
方式一：通过gv$ob_sql_audit 的数据来实现，不过这张视图是内存的数据，存在过期的问题，从这块获取的数据不全；
方式二：通过ocp的monitordb数据来实现，其中的ob_hist_sql_audit_sample_statistics(统计维度的慢SQL、并行SQL、长事务等数据), ob_hist_sql_audit_sample（采样的慢SQL、并行SQL、长事务等数据）, ob_hist_sqltext（SQL文本）
